lncfg/protocol_experimental_on.go

```diff
@@ -3,49 +3,7 @@
 
```

Comment:
think the experiment on and off files can also be removed now?

Reply:
@ellemouton and I had a little convo about this up here: https://github.com/lightningnetwork/lnd/pull/8744#discussion_r1597945128

Leaving it in sounds like the right move to me because looking at the commit history, the experimental protocol struct has housed other protocol features in the past, so it could be useful for future experimental flags in the future.